src/UhdmAst.cpp

```diff
@@ -744,7 +744,10 @@ AstNodeDType* getDType(FileLine* fl, vpiHandle obj_h, UhdmShared& shared) {
 }
 
 AstNode* process_operation(vpiHandle obj_h, UhdmShared& shared,
-                           const std::vector<AstNode*>& operands) {
+                           std::vector<AstNode*>&& operands) {
+    if(vpi_get(vpiReordered, obj_h))
```

Comment:
This is a nonstandard UHDM extension, it would be good to have a short comment when and why do we expect it here (ideally it would be described in UHDM itself, but it won't hurt to have more context).